core/web/schema/type/solana_key.graphql

```diff
@@ -0,0 +1,7 @@
+type SolanaKey {
+	publicKey: String!
```

Comment:
For caching usability you should returning an `id` here. In this case the ID can just be the public key

Reply:
👍🏽